contrib/cirrus/mac_env.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# This script is intended to be called by Cirrus-CI on a Mac M1 persistent workier.
+# It builds all the bits necessary for testing.
+
+# Many variables can affect operations, make them all known to assist debugging.
+echo "Selection of current env. vars:"
+for env_var_name in $(awk 'BEGIN{for(v in ENVIRON) print v}' | grep -Eiv '(^PATH$)|(^BASH_FUNC)|(^_.*)' | sort -u)
```

Comment:
I would hope that `-u` will never be needed (in `sort -u`) for envariables?!? As a casual reader this cost me time to figure out why it's there. Could you remove it to save someone else that time?